conda_forge_tick/auto_tick.py

```diff
@@ -849,7 +859,7 @@ def _compute_time_per_migrator(mctx, migrators):
         time_per_migrator[i] = time_per_migrator[i] * time_fac
 
     # recompute the total here
-    tot_time_per_migrator = sum(time_per_migrator)
+    tot_time_per_migrator = max(sum(time_per_migrator), 1)
```

Comment:
Can this also be zero?

Reply:
Yes. We divide by it later. It is only used for generating output IIRC. I'll move this call too.